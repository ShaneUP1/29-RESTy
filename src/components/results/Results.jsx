import React from 'react';
import PropTypes from 'prop-types';

const Results = ({ response }) => (
  <pre>
    {JSON.stringify(response, null, 2)}
  </pre>
);

Results.propTypes = {
  response: PropTypes.arrayOf(PropTypes.object).isRequired
};

export default Results;

