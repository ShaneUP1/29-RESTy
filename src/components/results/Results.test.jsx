import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Results from './Results';


describe('Results component', () => {
  afterEach(() => cleanup());
  it('renders a list of histories', () => {
    const { asFragment } = render(<Results
      results={[{}]}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
