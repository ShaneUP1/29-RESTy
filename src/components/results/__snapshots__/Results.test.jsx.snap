// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results component renders a list of histories 1`] = `
<DocumentFragment>
  <pre>
    [
  {}
]
  </pre>
</DocumentFragment>
`;
